sklearn/ensemble/forest.py

```diff
@@ -1485,6 +1570,15 @@ class ExtraTreesClassifier(ForestClassifier):
 
         .. versionadded:: 0.22
 
+    max_samples : int or float, default=None
+        If bootstrap is True, the number of samples to draw from X
+        to train each base estimator.
+            - If None (default), then draw `X.shape[0]` samples.
+            - If int, then draw `max_samples` samples.
+            - If float, then draw `max_samples * X.shape[0]` samples.
```

Comment:
I'm okay with the behavior of the float, but we need to document that here, i.e. explicitly say that if float, it must be in `(0, 1)`